### ISSUE TITLE
Feature Request: Total file size in use by backups locally and on drive

### ISSUE THREAD

**Issue by Spiffo** [sabeechen/hassio-google-drive-backup#139]
I think it would be good to be able to show on the Snapshots page, next to number of backups in Home Assistant and in Google Drive also the total space taken up by the backups. Perhaps this could also be exposed in a sensor.
![total storage used](https://user-images.githubusercontent.com/3015566/71969691-7a864080-3207-11ea-9a84-dd24f035c199.png)

**Comment by sabeechen** [comment]
This seems like a good idea to me.  I can also include the total storage usage as part of the attributes exposed from the existing "Snapshot State" sensor.   You'd have to make a template sensor to expose the value in HA, but that seems like a reasonable compromise since most users _probably_ won't care about it.

For a while I've also wanted to provide some kind of warning or notification if the addon can detect that creating another snapshot will exhaust the available storage.  It could only check optimistically at best because the snapshot size isn't really know until its completed and Hass.io will happily consume all available space on the disk trying to complete one, but I could just assume the next snapshot is going to be the same size as the most recent one which would at least catch most problems.  The addon runs inside a docker instance, so I'll have to see if disk usage statistics are accurately reflected inside the docker container.

Good suggestion!  I'm busy with other features right now, but I'll work this in eventually.

**Comment by sabeechen** [comment]
I've added this in the latest release v0.100.0.

**Comment by sheminasalam** [comment]
Hi I have seen the changes in the Web UI regarding the inclusion of size of the snapshots. It was also mentioned that the snapshot sensor will also output the size of the snapshots. I couldn't see that function. I was expecting it as I faced a problem very recently. There was some error with the Home assistant supervisor which lead to only partial backing up even when the option was set to full. This error caused my configuration.yaml to be left out of snapshots. I caught this only after a few days as I found that the snapshots were 100MB less than normal. So i was looking at for the proposed snapshot size sensor output to set a warning in such events.
